erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.py

```diff
@@ -306,5 +306,5 @@ def reconcile_dr_cr_note(dr_cr_notes, company):
 				}
 			]
 		})
-
+		jv.flags.ignore_mandatory = True
```

Comment:
@nemesis189 for which fields are we adding this? All mandatory fields are getting assigned some values right?